dts/bindings/wifi/infineon,airoc-wifi-common.yaml

```diff
@@ -0,0 +1,20 @@
+include: [base.yaml, pinctrl-device.yaml]
+
+properties:
+  wifi-reg-on-gpios:
+    description: |
+      Power-up/down gpio to control the internal regulators used
+      by the WiFi section of AIROC Wi-Fi device.
+    type: phandle-array
+
+  wifi-host-wake-gpios:
+    description: |
+      Host wake-up gpio. Signal from the AIROC Wi-Fi device
+      to the host indicating that the device requires attention.
+    type: phandle-array
+
+  wifi-dev-wake-gpios:
```

Comment:
can be named as `wakeup-gpios`  IMO

Reply:
Again, the name comes from the chip's datasheet.